### DISCUSSION TITLE
How to activate sub-images by default in xyz grid

### DISCUSSION THREAD

**Discussion by SAC020** [vladmandic/sdnext#1945] [Q&A] [answered]
Hi,

The default setting for xyz grid is to **not** generate sub-images. I keep forgetting about this and generate time consuming grids without the individual images. Is there a way to change this to generate sub-images by default?

Thanks!

**Comment by finefin** [comment, marked as answer]
open the file scripts/xyz_grid.py in a text editor of your choice.
search for the variable declaration in line 407 and change the value to true:

`include_lone_images = gr.Checkbox(label='Include Sub Images', value=True, elem_id=self.elem_id("include_lone_images"))`

This change will be gone if you update the script.

**Reply by SAC020** [reply]
Thank you!
